### ISSUE TITLE
LiCSBAS_plot_ts Error

### ISSUE THREAD

**Issue by xbdqs** [yumorishita/LiCSBAS#384]
Hi, Yumorishita
When I use the LiCSBAS_plot_ts.py to present my results, I meet the problem. I have tried to change the LiCSBAS package version as the LiCSBAS2 or COMET-version, but the issue still exits. What can I do?

The figs show the detail informatiom.
![issue](https://github.com/user-attachments/assets/b47bdda4-2de1-4a27-847d-e48163517b10)
![check](https://github.com/user-attachments/assets/2d8d9ba7-9320-4cd5-85d7-8cc9b97a3b01)

**Comment by yumorishita** [comment]
Maybe the processing was not successfully finished. Please show the full log.

**Comment by xbdqs** [comment]
Hi, Yumorishita,
This may be a bug in my system (caused by an external hard drive). Due to my step-by-step approach, I did not save any log records at the time. Today, I retested (with n=12) to conduct a quick test and saved the logs. The results showed that there were no issues with each processing step, and displaying the results using LiCBAS_plot_ts_py was also successful.

Thanks a lot for your reply, have a good day!
